src/python/CRABClient/Commands/tasks.py

```diff
@@ -67,3 +84,8 @@ def validateOptions(self):
                 self.date = date.today() - timedelta(days=days)
             except ValueError:
                 raise ValueError("\nPlease enter a NUMBER\nExample: crab tasks --days=10")
+        if self.options.status:
+            ## TODO: this shouldn't be hard-coded.
+            if self.options.status not in ['NEW', 'HOLDING', 'QUEUED', 'UPLOADED', 'SUBMITTED', 'RESUBMIT', 'FAILED', 'UNKNOWN']:
+                self.logger.info("\nPlease enter a valid STATUS\nExample: NEW, UPLOADED, SUBMITTED, FAILED\n")
+                sys.exit() 
```

Comment:
In other commands we use raise ConfigurationException(msg) when validation of an option fails. Can we do the same here?